package.json

```diff
@@ -5,6 +5,11 @@
   "description": "Adds a real-time collaborative whiteboard to Visual Studio Live Share sessions",
   "version": "0.0.11",
   "icon": "icon.png",
+  "extensionKind": [
```

Comment:
We don't need to add extensionKind anymore with recent vscode change. We do however need to add a `browser` property which would be the one used instead of `main`. i.e you need to add 

```
    "browser": "./dist/bundle.js",
```